src/java.base/share/classes/sun/nio/cs/ISO_8859_1.java

```diff
@@ -31,13 +31,17 @@
 import java.nio.charset.CharsetDecoder;
 import java.nio.charset.CharsetEncoder;
 import java.nio.charset.CoderResult;
-import java.util.Objects;
+import java.util.function.BiFunction;
 
 import jdk.internal.access.JavaLangAccess;
 import jdk.internal.access.SharedSecrets;
 import jdk.internal.util.Preconditions;
 import jdk.internal.vm.annotation.IntrinsicCandidate;
 
+import static java.util.Objects.requireNonNull;
+import static jdk.internal.util.Preconditions.AIOOBE_FORMATTER;
+import static jdk.internal.util.Preconditions.checkFromIndexSize;
```

Comment:
Avoid static imports of methods, it makes it harder to read and know where the methods are especially when crossing package boundaries.

Reply:
Changed as requested in 2672f7c1ada.